src/components/UserManagement/SetUpFinalDayButton.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Button } from 'reactstrap';
+import { SET_FINAL_DAY, CANCEL } from '../../languages/en/ui';
+import SetUpFinalDayPopUp from './SetUpFinalDayPopUp';
+import { updateUserFinalDayStatus } from 'actions/userManagement';
+import axios from 'axios';
+import { ENDPOINTS } from 'utils/URL';
+import { toast } from 'react-toastify';
+/**
+ * @param {*} props
+ * @param {Boolean} props.isBigBtn
+ * @param {*} props.userProfile.isSet
+ * @returns
+ */
+const SetUpFinalDayButton = (props) => {
+  const[isSet, setIsSet] = useState(false);  
+  const[finalDayDateOpen, setFinalDayDateOpen] = useState(false);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (props.userProfile?.endDate !== undefined)
+      setIsSet(true);
```

Comment:
Add braces around line 23, according to https://google.github.io/styleguide/jsguide.html#formatting-braces

Reply:
thx, move it in one line now